.github/renovate.json5

```diff
@@ -1,7 +1,11 @@
 {
   "extends": [
-    "config:recommended"
+    "config:recommended",
+    ":approveMajorUpdates",
```

Comment:
**1. Question:** I don't understand the [docs on `approveMajorUpdates`](https://docs.renovatebot.com/presets-default/#approvemajorupdates).
Does this mean we have to manually tick a checkbox inside https://github.com/GoogleCloudPlatform/microservices-demo/issues/728 to trigger major updates?

> Require dependency dashboard approval for `major` updates.

**2. Suggestion:** Are we able to exclude major versions from the 6 groupings you created instead? (so we automatically get major pull-requests, but via separate pull-requests)

Reply:
Update: Ah, I see that the groupings already exclude `major` updates. Nice!

Reply:
That said, I think we should embrace the automatic `major` updates
```suggestion
```